nntoolkit/create.py

```diff
@@ -35,6 +35,62 @@ def get_parser():
                         metavar="MODEL_FILE")
     return parser
 
+def xaviar10_weight_init(neurons_a, neurons_b):
+    fan_in = neurons_a
+    fan_out = neurons_b - 2
+    init_weight = 4.0*numpy.sqrt(6.0/(fan_in+fan_out))
+    _W = [numpy.random.uniform(low=-init_weight, high=init_weight, size=neurons_a) \
```

Comment:
Same goes for me, but pylint does not like single character variables. we
can write something like w_values as well.

2015-02-02 13:29 GMT+06:00 Martin Thoma <notifications@github.com>:

> In nntoolkit/create.py
> <https://github.com/MartinThoma/nntoolkit/pull/12#discussion_r23910083>:
>
> > @@ -35,6 +35,62 @@ def get_parser():
> >                          metavar="MODEL_FILE")
> >      return parser
> >
> > +def xaviar10_weight_init(neurons_a, neurons_b):
> > +    fan_in = neurons_a
> > +    fan_out = neurons_b - 2
> > +    init_weight = 4.0*numpy.sqrt(6.0/(fan_in+fan_out))
> > +    _W = [numpy.random.uniform(low=-init_weight, high=init_weight, size=neurons_a) \
>
> Why did you use an underscore? I usually use underscore prefixes for
> private attributes of objects.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MartinThoma/nntoolkit/pull/12/files#r23910083>.
>



-- 
<http://fiddle.jshell.net/sK5q2/1/show/index.htm>
Md. Sadaf Noor
www.sadafnoor.com
Student, BRAC School of Engineering and Computer Science,
 BRAC University, 66 Mohakhali, Dhaka, Bangladesh

 Head of IT, Kaan Pete Roi <http://www.shuni.org>, Dhaka
 Treasurer, ACM Chapter, <http://bracu.acm.org> BRAC University, Dhaka
Other current and past affiliations: | | | | In case of emergency or
failure to reach via this address, feel free to try contacting other
addresses. Other emails: sadafnoor@ieee.org, sadafnoor@acm.org,
email@sadafnoor.com | Mobile: +88 01758085078 | Website: www.sadafnoor.com
<http://sadafnoor.com/>